js/compiler/igor-controller.js

```diff
@@ -72,20 +79,43 @@ function job_remove(job) {
  * Select the flags for Igor to run the job.
  * @param {GMLProject} project
  * @param {string} runtime_path
+ * @param {string?} user_path
  * @param {IgorSettings} settings
  * @returns {Result<string[]>}
  */
-function job_flags_get(project, runtime_path, settings) {
+function job_flags_get(project, runtime_path, user_path, settings) {
+
+    /**
+     * @type {{[K in IgorPlatform]?: string}}
+    */
+    const exts = {
+        Windows: ".zip",
+        Mac: ".zip",
+        Linux: ".appimage",
+    };
+
+    let projectName = $gmedit['gml.Project'].current.name;
+    // Remove .yyp file extension
+    projectName = projectName.substring(0, projectName.length - 4);
+
     const flags = [
         `/project=${project.path}`,
         `/config=${settings.configName}`,
         `/rp=${runtime_path}`,
-        `/runtime=${settings.runtime}`
+        `/runtime=${settings.runner}`,
+        `/v`,
+        `/tf=${project.dir}/output/${projectName}${exts[igor_platform_cmd_name] ?? ""}`
     ];
+    if (user_path)
+        flags.push(`/uf=${user_path}`);
 
     switch (settings.verb) {
-        case 'Run':
         case 'Package':
+            if (['Windows', 'Mac'].includes(igor_platform_cmd_name))
```

Comment:
This is a style thing and applies for all single line `if`s - I avoid them in all of my projects so here I am enforcing that too, apologies if that wasn't clear, I wasn't really expecting anyone to contribute so have no style guide specified. The `{}` are mandatory!!